xml/System.Runtime.InteropServices/MemoryMarshal.xml

```diff
@@ -565,19 +578,19 @@ This method can be useful if part of a managed object represents a fixed array.
         <Parameter Name="value" Type="System.Byte*" Index="0" FrameworkAlternate="net-6.0;net-7.0" />
       </Parameters>
       <Docs>
-        <param name="value">The pointer to the null-terminated string of bytes.</param>
-        <summary>Creates a new read-only span for a null-terminated UTF8 string.</summary>
+        <param name="value">The pointer to the null-terminated narrow character string.</param>
+        <summary>Creates a new read-only span for a null-terminated narrow character string.</summary>
         <returns>A read-only span representing the specified null-terminated string, or an empty span if the pointer is <see langword="null" />.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
 
-The returned span does not include the `null` terminator, nor does it validate the well-formedness of the UTF8 data.
+This method is typically used with narrow character strings, such as ANSI or UTF-8 strings. The returned span does not include the `null` terminator, nor does it validate the well-formedness of the narrow character data.
```

Comment:
```suggestion
This method is typically used with narrow character strings, such as ANSI or UTF-8 strings. The returned span does not include the `null` terminator, nor does it validate whether the narrow character data is well-formed.
```

Reply:
Might be better for localization.

Reply:
I reworded this sentence entirely. Hopefully it's more clear now. :)